templates/real-time/index.html

```diff
@@ -2,317 +2,405 @@
-    <hr class="p-rule">
-    <div class="col">
-      <span class="p-text--small-caps">Video</span>
-      <h2>What is Real-time Ubuntu?</h2>
-    </div>
-    <div class="col">
-      <div class="u-embedded-media u-no-margin--bottom">
-        <iframe title="YouTube video player: What is Real-time Ubuntu?" class="u-embedded-media__element" src="https://www.youtube.com/embed/fxoWWfEWbIE?si=3Gi3EmZEhoQekrun" allowfullscreen=""></iframe>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <span class="p-text--small-caps">Video</span>
+        <h2>What is Real-time Ubuntu?</h2>
+      </div>
+      <div class="col">
+        <div class="u-embedded-media u-no-margin--bottom">
+          <iframe title="YouTube video player: What is Real-time Ubuntu?"
+                  class="u-embedded-media__element"
+                  src="https://www.youtube.com/embed/fxoWWfEWbIE?si=3Gi3EmZEhoQekrun"
+                  allowfullscreen=""></iframe>
+        </div>
+        <p>
+          Canonical and Intel collaborate to deliver real-time Ubuntu on 12th Gen Intel&reg; Core&trade; processors with TSN and TCC Support. Explore how real-time Ubuntu benefits the IoT customers and market as a whole.
+        </p>
       </div>
-      <p>Canonical and Intel collaborate to deliver real-time Ubuntu on 12th Gen Intel&reg; Core&trade; processors with TSN and TCC Support. Explore how real-time Ubuntu benefits the IoT customers and market as a whole.</p>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-section">
-  <div class="row--50-50">
-    <hr class="p-rule">
-    <div class="col">
-      <h2>Getting started</h2>
-    </div>
-    <div class="col">
-      <p>The real-time kernel is available via <a href="/pro">Ubuntu Pro</a>, Canonical’s enterprise security and compliance subscription. Ubuntu Pro is free for personal and small scale commercial use in up to 5 machines.</p>
-      <p>With an Ubuntu Pro subscription, launching the real-time kernel is as easy as:</p>
-      <pre><code>pro attach &lt;token&gt;
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h2>Getting started</h2>
+      </div>
+      <div class="col">
+        <p>
+          The real-time kernel is available via <a href="/pro">Ubuntu Pro</a>, Canonical's enterprise security and compliance subscription. Ubuntu Pro is free for personal and small scale commercial use in up to 5 machines.
+        </p>
+        <p>With an Ubuntu Pro subscription, launching the real-time kernel is as easy as:</p>
+        <pre><code>pro attach
 pro enable realtime-kernel</code></pre>
-      <a class="p-button" href="/pro">Discover Ubuntu Pro</a>
+        <a class="p-button" href="/pro">Discover Ubuntu Pro</a>
+        <p>
+          <a href="https://canonical-real-time-ubuntu-documentation.readthedocs-hosted.com/en/latest/">Read the documentation&nbsp;&rsaquo;</a>
```

Comment:
Change here

Reply:
According to the copy doc it should be on one line with the button